### PR TITLE
[Multiarch-1974,1972] Release Notes for OCP 4.10 for P/Z

### PR DESCRIPTION
**Pull request by alishaIBM**
Jira for P : https://issues.redhat.com/browse/MULTIARCH-1974
Jira for Z : https://issues.redhat.com/browse/MULTIARCH-1972

Draft for P/Z : 
https://docs.google.com/document/d/1LZqUJ86zmUt2Erh1mjhrACEbfBHEoE4z2Hgnr_MrugU/edit#

Reviewers :
For P : Manoj Kumar, Luvella
for Z : Holger Wolf, Florian Leber

Preview :
For P : https://deploy-preview-40806--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes.html#ocp-4-10-ibm-power

For Z : https://deploy-preview-40806--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes.html#ocp-4-10-ibm-z

Branch : enterprise-4.10
@SNiemann15

### CODE REVIEW

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -78,9 +78,162 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
...
+
+The following features are also supported on IBM Z and LinuxONE:
+
+* Currently, the following Operators are supported:
+** Cluster Logging Operator
+** Compliance Operator
+** NFD Operator
+** NMState Operator
+** OpenShift Elasticsearch Operator
+** Local Storage Operator
```
Comment (SNiemann15) [inline release_notes/ocp-4-10-release-notes.adoc:116]: @alishaIBM: Please move Local Storage Operator behind the Compliance Operator

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -78,9 +78,162 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
...
+
+The following new features are supported on IBM Z and LinuxONE with {product-title} {product-version}:
+
+* Compliance Operator
+* Horizontal pod autoscaling
+* The following Multus CNI plug-ins are supported: 
+** Bridge
+** Host-device
+** IPVLAN
+** IPAM
```
Comment (SNiemann15) [inline release_notes/ocp-4-10-release-notes.adoc:99]: Move IPAM before IPVLAN

File: release_notes/ocp-4-10-release-notes.adoc [hunk 3]
```diff
@@ -78,9 +78,162 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
...
+
+* NMState Operator
+* OVN-Kubernetes IPsec encryption
+* Vertical Pod Autoscaler
```
Comment (SNiemann15) [inline release_notes/ocp-4-10-release-notes.adoc:103]: Vertical Pod Autoscaler Operator

File: release_notes/ocp-4-10-release-notes.adoc [hunk 4]
```diff
@@ -78,9 +78,162 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
...
+The following features are also supported on IBM Z and LinuxONE:
+
+* Currently, the following Operators are supported:
+** Cluster Logging Operator
+** Compliance Operator
+** NFD Operator
+** NMState Operator
+** OpenShift Elasticsearch Operator
+** Local Storage Operator
+** Service Binding Operator
```
Comment (SNiemann15) [inline release_notes/ocp-4-10-release-notes.adoc:117]: Add: Vertical Pod Autoscaler Operator

File: release_notes/ocp-4-10-release-notes.adoc [hunk 5]
```diff
@@ -78,9 +78,162 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
...
+
+The following new features are supported on IBM Power Systems with {product-title} {product-version}:
+
+* Compliance Operator
+* Horizontal pod autoscaling
+* The following Multus CNI plug-ins are supported: 
+** Bridge
+** Host-device
+** IPVLAN
+** IPAM
```
Comment (SNiemann15) [inline release_notes/ocp-4-10-release-notes.adoc:180]: Move IPAM before IPVLAN

File: release_notes/ocp-4-10-release-notes.adoc [hunk 6]
```diff
@@ -78,9 +78,162 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
...
+
+* NMState Operator
+* OVN-Kubernetes IPsec encryption
+* Vertical Pod Autoscaler
```
Comment (SNiemann15) [inline release_notes/ocp-4-10-release-notes.adoc:184]: Vertical Pod Autoscaler Operator

File: release_notes/ocp-4-10-release-notes.adoc [hunk 7]
```diff
@@ -78,9 +78,162 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
...
+
+* Currently, the following Operators are supported:
+** Cluster Logging Operator
+** Compliance Operator
+** Local Storage Operator
+** NFD Operator
+** NMState Operator
+** OpenShift Elasticsearch Operator
+** SR-IOV Network Operator
+** Service Binding Operator
```
Comment (SNiemann15) [inline release_notes/ocp-4-10-release-notes.adoc:199]: Add: Vertical Pod Autoscaler Operator

**Comment by holgwolf** [comment]
LGTM

**Comment by lsmcfadden** [comment]
@manojnkumar can you input?

File: release_notes/ocp-4-10-release-notes.adoc [hunk 8]
```diff
@@ -78,8 +78,159 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
...
+* 4K FCP block device
+
+These features are available only for {product-title} on IBM Z and LinuxONE for {product-version}:
+
+* HyperPAV enabled on IBM Z and LinuxONE for the virtual machines for FICON attached ECKD storage
+
+[discrete]
+==== Restrictions
+
+Note the following restrictions for {product-title} on IBM Z and LinuxONE:
```
Comment (ahardin-rh) [inline release_notes/ocp-4-10-release-notes.adoc:139]: Could we reword this to omit "note"? Perhaps: 
The following restrictions impact {product-title} on IBM Z and LinuxONE:

File: release_notes/ocp-4-10-release-notes.adoc [hunk 9]
```diff
@@ -78,8 +78,159 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
...
+* Persistent storage using Raw Block
+* Support for multiple network interfaces
+* Support for Power10
+* Three-node cluster support
+* 4K Disk Support
+
+[discrete]
+==== Restrictions
+
+Note the following restrictions for {product-title} on IBM Power:
```
Comment (ahardin-rh) [inline release_notes/ocp-4-10-release-notes.adoc:218]: Same here. Could we reword this to omit "note"? Perhaps:
The following restrictions impact {product-title} on IBM Power:
